File: csharp/item/InsertItem.cs
using System;


namespace SdkSample.item
{
    public class InsertItem
    {
        public static void Execute()
        {
            try
            {
                string itemId = ItemFunctions.InsertSampleItem();
                ItemFunctions.DeleteSampleItem(itemId);
            }
            catch (Exception e)
            {
                Console.WriteLine("An Exception occurred. Please review the following error:");
                Console.WriteLine(e.ToString()); // handle gracefully
                Environment.Exit(1);
            }
        }
    }
}

File: java/src/item/InsertItem.java
package item;

public class InsertItem {
   public static void execute() {
       try {
           String itemId = ItemFunctions.insertSampleItem();
           ItemFunctions.deleteSampleItem(itemId);
       }
       catch (Exception e) {
           System.out.println("An Exception occurred. Please review the following error:");
           System.out.println(e.toString()); // handle gracefully
           System.exit(1);
       }
   }
}